legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/routing/router_routing.pure

```diff
@@ -626,6 +627,8 @@ function meta::pure::router::routing::shouldStop(f:Function<Any>[1], extensions:
 function meta::pure::router::routing::shouldStopFunctions(extensions:meta::pure::extension::Extension[*]):Function<Any>[*]
 {
   [
+    // TODO HSO: Is this right
+    internalize_Class_1__Binding_1__String_1__T_MANY_,
```

Comment:
This seems odd to me, as internalize is a function we already support.  @hardikmaheshwari - what are your thoughts?

Reply:
Will look into if this is still needed.